composites/SearchResultForm/SearchResultForm.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import { localize } from "i18n-calypso";
+import PropTypes from "prop-types";
 
+import { localize } from "../../utils/i18n";
```

Comment:
I did a search and replace for all occurences of `import { localize } from "i18n-calypso";`, it doesn't seem necessary indeed.